src/wiki/plugins/images/templatetags/wiki_thumbnails.py

```diff
@@ -0,0 +1,2 @@
+from __future__ import absolute_import
```

Comment:
Just verified it, this *is* actually the reason :) I think it's to provide a decent wrapper for sorl.thumbnail on case it changes API or we add an alternative. Or maybe it's there for some historic reason.